ionos_sbx/wallet/vault/templates/cronjob.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-check-cronjob
```

Comment:
Is it possible to use literal vallue here instead of variable?

Reply:
Yes, now use a literal value.